docs/tutorials/fundamentals/tutorial-obtain-token-and-purchase-ram.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Tutorial - Using the Faucet and Buying RAM on Ultra Testnet'
+order: -99996
+outline: [0, 5]
+---
+
+# Tutorial - Using the Faucet and Buying RAM on Ultra Testnet
+
+This tutorial will guide you through the process of obtaining UOS tokens using the faucet and purchasing RAM on the Ultra blockchain testnet. These actions are crucial for participating in blockchain activities, such as deploying contracts or storing unique digital assets (Uniqs).
+
+## Prerequisites
+
+Before starting this tutorial, ensure you have the following:
+
+- **A Developer Testnet Account**: You need an account on the Ultra testnet. If you don't have one, please refer to our previous tutorial on [how to create a testnet account](./tutorial-generate-key-and-create-testnet-account.md).
+- **Private and Public Keys**: Your account’s keys should be securely stored. You’ll need your private key for signing transactions during this tutorial.
```

Comment:
This point should be about having Wallet installed and having your key added there, since that's how they will be using their private key for signing